src/jdk.jpackage/macosx/classes/jdk/jpackage/internal/MacFromParams.java

```diff
@@ -185,6 +209,36 @@ private static MacDmgPackage createMacDmgPackage(
         return pkgBuilder.create();
     }
 
+
+    // JDK bundle: "Contents/Home", "Contents/MacOS/libjli.dylib"
+    // and "Contents/Info.plist"
+    private static boolean isRuntimeImageJDKBundle(Path runtimeImage) {
+        Path path1 = runtimeImage.resolve("Contents/Home");
+        Path path2 = runtimeImage.resolve("Contents/MacOS/libjli.dylib");
+        Path path3 = runtimeImage.resolve("Contents/Info.plist");
+        return IOUtils.exists(path1)
+                && path1.toFile().list() != null
+                && path1.toFile().list().length > 0
+                && IOUtils.exists(path2)
+                && IOUtils.exists(path3);
```

Comment:
Please don't use `IOUtils.exists()` and other redundant wrappers over the `Files` class in the new code. Use `Files.exists()` instead.